docs/goodtoknow.rst

```diff
@@ -43,3 +43,36 @@ Which function to use when resampling data
 	to go from daily data to coarser (monthly, seasonal, annual).
 
 What is currently not covered by either `xscen` or `xclim` is a method to resample from data coarser than daily, where the base period is non-uniform (ex: resampling from monthly to annual data, taking into account the number of days per month).
+
+
+Metadata translation
+--------------------
+
+xscen itself does not add many translatable attributes, but when it does, it will look into xclim's options for which locales to translate them to. In a config file, activating French translations for both xclim's indicators and xscen is done with :
+
+.. code-block:: yaml
+
+	xclim:
+		metadata_locales:
+		  - fr
+
+Note that this only applies to attributes that are added to a dataset. Some xscen functions will instead update an existing attribute. For example, when calculating the climatology of a variable with `long_name` `Mean temperature`, :py:func:`climatological_mean` will update the `long_name` as `30-year average of Mean temperature`. This automatic update is done for all locales available in the variable, no matter what xclim option is activated.
```

Comment:
not clear to me from this sentence that we will also modify a `long_name_fr` with `moyenne de 30 ans de la température moyenne` ..